SynLog.pas

```diff
@@ -2243,8 +2243,10 @@ class procedure TSynMapFile.Log(W: TTextWriter; aAddressAbsolute: PtrUInt;
     exit;
   {$ifdef FPC}
   s := BacktraceStrFunc(pointer(aAddressAbsolute));
-  if Pos('SynLog.pas',s)=0 then // don't log internal calls
+  if Pos('SynLog.pas',s)=0 then begin // don't log internal calls
     W.AddShort(s);
+    W.AddShort(#14);

```

Comment:
what is this #14 for?

Reply:
this is lifehack to break stacktrace in viewer -- I use it to break call stack to separate lines. Don''t know but #10 didnt worked.
![ttt](https://user-images.githubusercontent.com/11839048/56931313-da0c4800-6adf-11e9-8ccd-fa0f35d84b80.png)